### PR TITLE
Making sponsors page of documentation

### PR DESCRIPTION
**Pull request by smithis7**
Added a sponsors page to the documentation as can be seen [here](https://smithis7.github.io/tardis/branch/sponsor_doc/sponsors.html).

**Description**

The sponsors page includes logos for Google, ESA, and NumFOCUS.

**Motivation and context**

Wolfgang requested that this page be added to the documentation.

**How has this been tested?**
- [ ] Testing pipeline.
- [x] Other. 
Documentation build locally and on github.

**Type of change**

- [ ] Bug fix. 
- [x] New feature. 
- [ ] Breaking change. 
- [ ] None of the above. 

**Checklist**

- [x] My change requires a change to the documentation.
    - [x] I have updated the documentation accordingly.
    - [x] (optional) I have built the documentation on my fork following [the instructions](https://tardis-sn.github.io/tardis/development/documentation_preview.html).
- [x] I have assigned and requested two reviewers for this pull request.

### CODE REVIEW

File: docs/sponsors.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
+********
+Sponsors
+********
+
```
Comment (jaladh-singhal) [inline docs/sponsors.rst:4]: Can you add a one-line explanation, something like "We're grateful to the following organisations for sponsoring us and helping us sustain TARDIS as a free and open source software."?

**Review by jaladh-singhal — CHANGES_REQUESTED** [review]
Look great - thanks @smithis7 for making this so quick!

**Review by jaladh-singhal — APPROVED** [review]

**Comment by Molkree** [comment]
I don't see the contents of this page in Chrome, not even the `We are grateful to the following organisations` line.

image

It's just fine in Edge.

image

**Comment by smithis7** [comment]
> I don't see the contents of this page in Chrome, not even the `We are grateful to the following organisations` line.
> 
> image
> 
> It's just fine in Edge.
> 
> image

That's very odd. It works fine for me in chrome. Does the page source appear correctly when you click "view page source" in chrome?

**Comment by Molkree** [comment]
@smithis7, it seems so?
image

But the contents are 0px height
image

**Comment by Molkree** [comment]
Okay, I see why it happens, it's AdGuard.

**Comment by Molkree** [comment]
For anyone interested, AdGuard has [updated](https://github.com/AdguardTeam/AdguardBrowserExtension/issues/1815) filters so it works correctly now.

**Pull request merged**